ehsm_kms_service/secret_manager_apis.js

```diff
@@ -265,9 +314,124 @@ const createSecret = async (res, appid, payload, DB) => {
     }
 }
 
+//Force delete secret or schedule a time to delete secret
```

Comment:
I'd like to suggest adding parameter descriptions of payload for each secret manager's APIs.
secretName       Required    Length(1~100)    
recoveryPeriod  "brief introduction of the item if it's not straightforward"
Optional     Range(0~365)  Uint(ms/s/min/hour/day/month?)  DefaultValue(30d)   
forceDelete       "xxxxxxxx"
Optional    Range(T/F)        DefaultValue(False)

Reply:
will be add description for function